src/files/FilesPage.js

```diff
@@ -357,7 +357,11 @@ const FilesPage = ({
 
       <MainView t={t} files={files} remotePins={remotePins} pendingPins={pendingPins} failedPins={failedPins} doExploreUserProvidedPath={doExploreUserProvidedPath}/>
 
-      <Preview files={files} onDownload={() => onDownload([files])} />
+      <Preview files={files} onDownload={() => onDownload([files])} onClose={() => {
```

Comment:
Also, this should use `useCallback`.. coming in my fix.

Reply:
fixed by [`9c6c697` (#2401)](https://github.com/ipfs/ipfs-webui/pull/2401/commits/9c6c6973610cc406525b11fec2357f7c348c8419)